allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
no, the problem is how long-line looks when there is line breaks. So when there is possibility to see the multiline, it should increase line-height. In other cases line height don't plays big role. 

As for envs - they can be breaked and here the line height adds enough space.
![image](https://cloud.githubusercontent.com/assets/1964214/15504335/24e350c2-21c7-11e6-94d9-19a0f3849e03.png)